lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -156,6 +156,13 @@
     - "vmware-tools needs to be installed on given virtual machine in order to work with this parameter."
     default: 'no'
     type: bool
+  wait_for_state_change:
```

Comment:
If this is not a boolean, the name should be more clear (`wait_for_state_change` sounds like a true/false value).

Suggest to make it `state_change_timeout`, description can stay